src/document.c

```diff
@@ -1545,6 +1564,21 @@ gboolean document_reload_force(GeanyDocument *doc, const gchar *forced_enc)
 	pos = sci_get_current_position(doc->editor->sci);
 	new_doc = document_open_file_full(doc, NULL, pos, doc->readonly, doc->file_type, forced_enc);
 
+	if (file_prefs.keep_edit_history_on_reload && file_prefs.show_keep_edit_history_on_reload_msg)
+	{
+		bar = document_show_message(doc, GTK_MESSAGE_INFO,
+						on_keep_edit_history_on_reload_response,
+						GTK_STOCK_OK, GTK_RESPONSE_ACCEPT,
+						"Discard history", GTK_RESPONSE_NO,
+						NULL, 0, "You can revert the buffer to the previous state "
```

Comment:
I disagree but well, I'll change it.

But remember that this line appears in the GUI directly under the headline "The file has been reloaded." so it's sufficiently clear IMO.